android/app/src/main/res/values-fr/strings.xml

```diff
@@ -848,12 +848,19 @@
 	<string name="type.amenity.parking.private">Stationnement privé</string>
 	<string name="type.amenity.parking.park_and_ride">Parking relais</string>
 	<string name="type.amenity.parking.underground">Stationnement souterrain</string>
-	<string name="type.amenity.parking.street_side">Stationnement</string>
-	<string name="type.amenity.parking.fee.street_side">Stationnement</string>
-	<string name="type.amenity.parking.lane">Stationnement</string>
-	<string name="type.amenity.parking.fee.lane">Stationnement</string>
+	<string name="type.amenity.parking.fee.underground">Parking souterrain payant</string>
+	<string name="type.amenity.parking.underground.private">Parking privé souterrain</string>
+	<string name="type.amenity.parking.private.underground">Parking privé souterrain</string>
+	<string name="type.amenity.parking.street_side">Stationnement sur le côté de la rue</string>
+	<string name="type.amenity.parking.fee.street_side">Payé parking côté rue</string>
```

Comment:
```suggestion
	<string name="type.amenity.parking.fee.street_side">Rue à stationnement payant</string>
```

Reply:
Thanks! Next time is better if you make the translation suggestions to the `data/strings/types_strings.txt` file, as these other files are generated by a script. Makes it easier to apply the changes!